app/assets/stylesheets/donor.css.scss

```diff
@@ -236,3 +236,12 @@
     font-weight: 500;
   }
 }
+
+.best_in_place:hover {
+    cursor: pointer;
+    color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces